score/score.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
This kind of information should probably just be part of the `Model` object to allow for filtering when doing experiments. The hierarchy doesn't feel compelling to me because already there are a few shoehorns: 'recommendation, recommendation'. 'other, other task'. Perhaps just add a `keywords` field to the Model objects?